### DISCUSSION TITLE
[Kavita] Allow multiple users per email

### DISCUSSION THREAD

**Discussion by neroes** [Kareadita/Kavita#3432] [Ideas]
### Idea Description

Allow multiple users to share an email

This would be nice to have when you are the main admin, then you have one account to do administrative stuff, and one for use with limited access rights
if your device is stolen or otherwise compromised then it is more likely to be the one for use

### Idea Category

Feature Enhancement

### Duration of Using Kavita

a few months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by scare376** [comment]
I don't see the use case for this. It sounds like you want to overcomplicate the user accounts for little to no gain. Just use multiple emails or you can use Gmail (other services might have the same or something similar) will let you add to their email with + and it will still direct it to the same address.

**Reply by scare376** [reply]
You could also forgo the email entirely
